src/views/account-details/components/transaction-amount/transaction-amount.view.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { TxType } from "@hermeznetwork/hermezjs/src/enums";
+
+import FiatAmount from "src/views/shared/fiat-amount/fiat-amount.view";
+
+interface TransactionAmountProps {
+  type: TxType;
+  preferredCurrency: string;
+  accountIndex: string;
+  fiatAmount?: number;
+  fromAccountIndex?: string;
+}
+
+function TransactionAmount({
+  type,
+  fiatAmount,
+  preferredCurrency,
+  fromAccountIndex,
+  accountIndex,
+}: TransactionAmountProps): JSX.Element {
+  const negativeFiatAmount = fiatAmount && -fiatAmount;
```

Comment:
I would add a prop to the `FiatAmount` to display a negative number instead of creating a negative number.

Reply:
We've decided to remove any negative number